model_analyzer/result/result_table_manager.py

```diff
@@ -0,0 +1,609 @@
+        Utility function that creates a table with column
+        headers corresponding to perf_analyzer arguments
+        and requested metrics. Also sets the result
+        comparator for that table.
+        """
+
+        # Create headers
+        self._result_tables[table_key] = ResultTable(headers=headers,
+                                                     title=title)
+
+    def _tabulate_measurements(self, run_config_result):
+        """
+        checks RunConfigMeasurements against constraints,
+        and puts them into the correct (passing or failing)
+        table
+        """
+
+        model_name = run_config_result.model_name()
+        instance_groups, max_batch_sizes, dynamic_batchings, cpu_onlys, backend_parameters = self._tablulate_measurements_setup(
+            run_config_result)
+
+        passing_measurements = run_config_result.passing_measurements()
+        failing_measurements = run_config_result.failing_measurements()
+
+        for (run_config_measurements, passes) in [(passing_measurements, True),
+                                                  (failing_measurements, False)
+                                                 ]:
+            for run_config_measurement in run_config_measurements:
+                self._tabulate_measurement(
+                    model_name=model_name,
+                    instance_groups=instance_groups,
+                    max_batch_sizes=max_batch_sizes,
+                    dynamic_batchings=dynamic_batchings,
+                    run_config_measurement=run_config_measurement,
+                    passes=passes,
+                    cpu_onlys=cpu_onlys,
+                    backend_parameters=backend_parameters)
+
+    def _tablulate_measurements_setup(self, run_config_result):
+        model_configs = [
+            model_run_configs.model_config() for model_run_configs in
+            run_config_result.run_config().model_run_configs()
+        ]
+        instance_groups = [
+            model_config.instance_group_string()
+            for model_config in model_configs
+        ]
+        max_batch_sizes = [
+            model_config.max_batch_size() for model_config in model_configs
+        ]
+        dynamic_batchings = [
+            model_config.dynamic_batching_string()
+            for model_config in model_configs
+        ]
+        cpu_onlys = [
+            run_config_result.run_config().cpu_only()
+            for model_config in model_configs
+        ]
+        backend_parameters = [
+            model_config._model_config.parameters
+            for model_config in model_configs
+        ]
+
+        return instance_groups, max_batch_sizes, dynamic_batchings, cpu_onlys, backend_parameters
+
+    def _tabulate_measurement(self, model_name, instance_groups,
+                              max_batch_sizes, dynamic_batchings,
+                              run_config_measurement, passes, cpu_onlys,
+                              backend_parameters):
```

Comment:
Hard to parse. Can we list arguments line by line as in `_get_common_row_items`?

Reply:
The formatter dictates this.